baseapp/extractor_utils.go

```diff
@@ -7,6 +7,21 @@ import (
 	"os"
 )
 
+//Denominations represents the list of all denominations at a block height
+type Denominations = []string
+
+var genesisDenominations = map[string]Denominations{
+	//TODO : Find genesis denominations for all chains and update this list
+
+	"columbus-1": []string{"uatom"},
```

Comment:
this will never by uatom, but uluna here. are we able to interrogate the genesis file directly (either through the app, or worst case load it here and parse the relevant fields?)